static/js/protein/chart.ts

```diff
@@ -442,10 +442,10 @@ export function drawProteinInteractionChart(
     .attr("height", y.bandwidth())
     .style("fill", BAR_COLOR)
     //PROTEIN_REDIRECT
-    .on("click", function(d) {
+    .on("click", function (d) {
       const protein_url = "bio/" + d.name + "_" + d.parent;
```

Comment:
- variable names should be camelCase
- maybe proteinId instead proteinUrl?

Reply:
Done!